templates/base.html

```diff
@@ -1,34 +1,29 @@
 {% load static %}
```

Comment:
Was the reverting of base back to base-deprecated intentional? If so then let's have a discussion on what we want the look of the site to be.

Reply:
It was not a reversion. I just removed base-deprecated and updated base.html with what it needed to support both